aikau/src/main/resources/alfresco/renderers/Progress.js

```diff
@@ -344,7 +344,10 @@ define(["dojo/_base/declare",
          filesAdded = parseInt(filesAdded, 10);
          totalFiles = parseInt(totalFiles, 10);
 
-         if (!total || !done || !filesAdded || !totalFiles)
+         // 0 is a legal numeric value (e.g. initial "start" progress)
+         // can't use simple truthy vs non-truthy check
+         if (isNaN(total) || total < 0 || isNaN(done) || done < 0 ||
+                 isNaN(filesAdded) || filesAddded < 0 || isNaN(totalFiles) || totalFiles < 0)
```

Comment:
I think the issue here is a simple typo... `filesAddded` needs to be `filesAdded`.